dags/glam_glean_imports.py

```diff
@@ -71,7 +71,7 @@
 )
 
 # Move logic from Glam deployment's GKE Cronjob to this dag for better dependency timing
-glam_import_image = 'gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2021.8.1-10'
+glam_import_image = 'gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2022.02.0-15'
```

Comment:
how were you able to determine this value? is this something we have to manually update every time there's a change to our etl script?

Reply:
retrieved it from here :https://console.cloud.google.com/gcr/images/moz-fx-dataops-images-global/global/gcp-pipelines%2Fglam%2Fglam-production%2Fglam@sha256:dec9490a7b1acb25e2f3fe07a21455bc3747f85bae9204cd34a1ddaab189d7b9/details?tab=vulnz

Reply:
what was the reason for the last generated image? is it for the new GLAM production release we did this week?